LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2021 Splunk Inc.
+   Copyright 2022 Splunk Inc.
```

Comment:
@rgil-splunk Should it be 2021-2022 or just 2022 is fine?

Reply:
Somehow, in LICENSE file, it has only 1 year. In NOTICE file, it may have 1 year or range year and may not consistent with other files. Do we need to fix this? Well, several files have been merged to next.